### ISSUE TITLE
 When I launch the editor on google colab, the server is disconnected.

### ISSUE THREAD

**Issue by mule-engineer13** [nerfstudio-project/nerfstudio#1085]
Hi nerfstudio Team!
I would appreciate it if you could answer one question.

I want to try the demo below but it doesn't work.
https://colab.research.google.com/github/nerfstudio-project/nerfstudio/blob/main/colab/demo.ipynb

I ran up to "Set up and Start Viewer" and launched the Viewer, but it says "Server Disconnected".
If you know the cause and solution, please let me know.

**Comment by mule-engineer13** [comment]
Yes, after running the  following training cell and trying to start the viewer again I get a "server disconnected" error.

**Comment by amir-opal** [comment]
I have the same problem. Would you tell us whether it can be solved and we can use nerfstudio on colab?

**Comment by puzzlefish** [comment]
Commenting out the condalab installation worked for me.

See my fix for #1077 and [my quick fix](https://github.com/puzzlefish/nerfstudio_demo_issue_1077/blob/main/nerfstudio_demo_issue_1077.ipynb).

In detail: From what I understand, the "server disconnected" error shows because the training failed, which is because the tiny cuda installation failed. This is in turn due to the fact that the tiny cuda installation depends on a wheel compiled for Python 3.7, but Google Colab recently switched to Python 3.8. Simply reverting to the fallback runtime version (Tools > Command palette > Use fallback runtime version) doesn't work because the condacolab installation restarts the runtime, hence falling back to the default Python 3.8.
